apps/explorer/views.py

```diff
@@ -155,6 +163,60 @@ def post(self, request, *args, **kwargs):
         return super().post(request, *args, **kwargs)
 
 
+class PixelSetUnselectView(LoginRequiredMixin, FormView):
+
+    form_class = SessionPixelSetSelectForm
+    http_method_names = ['post', ]
+    success_url = reverse_lazy('explorer:pixelset_list')
```

Comment:
Shouldn't we return to the exact previous url instead?

Reply:
Damned. You are right.